src/sss_client/pam_sss_gss.c

```diff
@@ -21,6 +21,7 @@
 #include <stdlib.h>
 #include <stddef.h>
 #include <stdbool.h>
+#include <security/pam_appl.h>
```

Comment:
Why is this change required?
Function `pam_get_item()` is declared in `security/_pam_types.h` which is included by `security/pam_appl.h` as you noticed, but also by `security/pam_modules.h` which is included from the next line.